runtime/src/accounts.rs

```diff
@@ -862,17 +862,17 @@ impl Accounts {
 
     /// Only called from startup or test code.
     #[must_use]
-    pub fn verify_bank_hash_and_lamports(
+    pub fn verify_accounts_hash_and_lamports(
         &self,
         slot: Slot,
         total_lamports: u64,
         config: BankHashLamportsVerifyConfig,
```

Comment:
maybe the struct should also be renamed to `AccountsHashLamportsVerifyConfig`?

Reply:
similarly, the error type `BankHashVerificationError` is used in the accounts_db function. If it's only used for verifying `AccountsHash`, let's change its' name too?